plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java

```diff
@@ -1096,6 +1099,45 @@ public void testParquetShortDecimalWriteToTrinoBigintBlockWithNonZeroScale()
                 .isInstanceOf(TrinoException.class);
     }
 
+    @Test(dataProvider = "timestampPrecision")
+    public void testTimestamp(HiveTimestampPrecision precision)
+            throws Exception
+    {
+        List<Long> epochMillisValues = ContiguousSet.create(Range.closedOpen((long) -1_000, (long) 1_000), DiscreteDomain.longs()).stream()
+                .map(millis -> System.currentTimeMillis() + millis)
+                .collect(toImmutableList());
+        List<Timestamp> writeValues = epochMillisValues.stream()
+                .map(AbstractTestParquetReader::longToTimestamp)
+                .collect(toImmutableList());
+        List<SqlTimestamp> readValues = epochMillisValues.stream()
+                .map(epochMillis -> SqlTimestamp.newInstance(precision.getPrecision(), epochMillis * 1_000, 0))
+                .collect(toImmutableList());
+        // INT96 backed timestamps are written by default
```

Comment:
By default in Hive connector, or by default in Parquet writer?

"default" suggests there is a way to choose a different behavior, but it seems there isn't, right?
-> `timestamps are written using int96 physical type` ?

Reply:
I meant default ParquetSchemaOptions here, will update comment